deviceinfo.js

```diff
@@ -255,93 +275,103 @@ const devicesWithNotch = [
 
 const deviceNamesByCode = {
   'iPod1,1': 'iPod Touch', // (Original)
-  'iPod2,1': 'iPod Touch', // (Second Generation)
```

Comment:
do not propose whitespace changes please, they are essentially without value and just make review and attribution harder